### ISSUE TITLE
patch version support (e.g. 7.9.2)

### ISSUE THREAD

**Issue by jjangga0214** [blacktop/docker-elasticsearch-alpine#38]
Hi!

Does this cool alpine repo intentionally not support patch versions?

For example, there are 7.9.1 and 7.9.2 in [the official elastic repo](https://hub.docker.com/_/elasticsearch?tab=tags).

Thanks.

**Comment by blacktop** [comment]
https://hub.docker.com/r/blacktop/elasticsearch/tags

**Comment by jjangga0214** [comment]
@blacktop Oh, I see. 
I first thought there were no PATCH versions as README only lists MINOR versions.
To prevent confusion, how about adding an sentence like this?
```
PATCH versions are also supported. For example, 7.9.1 is available along with 7.9. 
```

**Comment by blacktop** [comment]
So it is the docker standard that the tags (for example) :7 :7.9 and :7.9.2 (also no tag which is the same as :latest) are all identical meaning if you want to pull the latest you use any of those. This is how all the official images work and how this image works. We tag all the patches too so if for some reason you need 7.9.1 you can still grab it. As it is the standard we don't feel the need to explicitly state it.
